### PR TITLE
add more visual tests

### PR DESCRIPTION
**Pull request by achamas-playco**
### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 3a8b5d2</samp>

This pull request adds, updates, and fixes several test scenes for rendering graphics, sprites, and text with the `pixijs` library. It demonstrates how to use different classes, properties, styles, filters, and textures to create various shapes, effects, and texts. It also improves the code quality and consistency of the test scenes by removing unused imports and parameters, renaming files and descriptions, and adding imports as needed. The new and modified files are `lines-arc.scene.ts`, `sprite.scene.ts`, `bitmap-text.scene.ts`, `text.scene.ts`, `nested-container.scene.ts`, `circle.scene.ts`, `rect-fills.scene.ts`, `rect.scene.ts`, and `shared-context.scene.ts`.

### CODE REVIEW

File: tests/visual/snapshots/webgpu-sprite-scene-ts.png
Comment (Zyie) [inline tests/visual/snapshots/webgpu-sprite-scene-ts.png:1]: this new image seems to be empty?

File: tests/visual/snapshots/webgpu-tiling-sprite-scene-ts.png
Comment (Zyie) [inline tests/visual/snapshots/webgpu-tiling-sprite-scene-ts.png:1]: same with this one

**Review by Zyie — CHANGES_REQUESTED** [review]

**Comment by bigtimebuddy** [comment]
Could we please not use copyright/trademarked logos (e.g., playco's logo)?

**Comment by achamas-playco** [comment]
@GoodBoyDigital @Zyie I've tried multiple things to get this build to pass, it passes locally. I suspect it's using the snapshots of `next-v8` or something else other than the ones on this branch. I deleted all snapshots temporarily and that commit passed. When I added another commit with the snapshots freshly generated it failed. We might need to delete the snapshots, disable visual tests temporarily, and merge that, then raise a PR adding back the snapshots. Thoughts?

**Comment by Zyie** [comment]
> @GoodBoyDigital @Zyie I've tried multiple things to get this build to pass, it passes locally. I suspect it's using the snapshots of `next-v8` or something else other than the ones on this branch. I deleted all snapshots temporarily and that commit passed. When I added another commit with the snapshots freshly generated it failed. We might need to delete the snapshots, disable visual tests temporarily, and merge that, then raise a PR adding back the snapshots. Thoughts?

The CI does sometimes render these tests slightly differently, in these cases we just increase the `pixelMatch` value to higher. I've updated these for you and now everything is passing!

You can also check the output of the CI in the summary section and download the `pr_uploads` zip: https://github.com/pixijs/pixijs/actions/runs/7112704608

**Review by Zyie — APPROVED** [review]

**Pull request merged**